<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DatiInverter</class>
 <widget class="QDialog" name="DatiInverter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>851</width>
    <height>304</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Selezione inverter</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Cerca</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineSearch"/>
     </item>
     <item>
      <widget class="QComboBox" name="comboFilter">
       <item>
        <property name="text">
         <string>Efficienza</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Vmp</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Pnom</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Costruttore</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Modello</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Vmin</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Vmax</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Vmpmin</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Vmpmax</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>tipo</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Marca</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Dati</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QListView" name="listMarca">
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QTableView" name="tViewDatiInverter">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushApplica">
       <property name="text">
        <string>Applica</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushElimina">
       <property name="text">
        <string>Elimina</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DatiInverter</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DatiInverter</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
